contrib/ibm-mq/containersource.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: ContainerSource
+metadata:
+  name: ibm-mq
+spec:
+  template:
+    spec:
+      containers:
+        - image: knative.dev/eventing-contrib/contrib/ibm-mq
+          name: receiver
+          env:
+            - name: LD_LIBRARY_PATH
+              value: /var/run/ko/lib64
+            - name: QUEUE_MANAGER
+              value: QM1
+            - name: CHANNEL_NAME
+              value: DEV.APP.SVRCONN
+            - name: CONNECTION_NAME
+              value: mq-service.default.svc.cluster.local(1414)
+            - name: QUEUE_NAME
+              value: DEV.QUEUE.1
+            - name: USER_ID
+              value: <USER_ID>
+            - name: PASSWORD
+              value: <PASSWORD>
+  sink:
+    apiVersion: eventing.knative.dev/v1alpha1
+    kind: Channel
```

Comment:
Given the switch to brokers/triggers, should this be "Broker" instead?  I'm still new to the broker/channel split so it might be a silly question.

Reply:
Not sure about that. Changing type to `Broker` won't be enough to make it work as broker I guess.

Reply:
Perhaps @n3wscott can comment on this?  Scott - which would make most sense, Broker or Channel?